readthedocs/projects/tasks.py

```diff
@@ -722,14 +724,19 @@ def setup_vcs(self, environment):
         if commit:
             self.build['commit'] = commit
 
-    def get_env_vars(self):
-        """Get bash environment variables used for all builder commands."""
+    def get_rtd_env_vars(self):
+        """Get bash environment variables specific to Read the Docs."""
         env = {
-            'READTHEDOCS': True,
+            'READTHEDOCS': 'True',
```

Comment:
Is this backwards compatible? Seems like it will change what users might be checking again?

Reply:
Yes, actually this is an error when building in a local environment, but we didn't see it bc the local build env was overriding all these. Env vars are a dict of string: string